src/components/atoms/Timeline/ScaleList.tsx

```diff
@@ -0,0 +1,117 @@
+import { memo } from "react";
+
+import { styled } from "@reearth/theme";
+import { XXSRegular } from "@reearth/theme/fonts";
```

Comment:
Sorry 99%-100% of the time please use anything related to fonts through the Text atom. Then you can do like `<Text size="xxs">TEXT HERE</Text>`

Reply:
🙏 [refactor: use Text component](https://github.com/reearth/reearth-web/pull/264/commits/4c7da4da79cfbb23660b5ba4d7ea3edeb27e142f)